src/mpid/ch4/src/mpidig_part_utils.h

```diff
@@ -2,12 +2,38 @@
  * Copyright (C) by Argonne National Laboratory
  *     See COPYRIGHT in top-level directory
  */
-
 #ifndef MPIDIG_PART_UTILS_H_INCLUDED
 #define MPIDIG_PART_UTILS_H_INCLUDED
 
 #include "ch4_impl.h"
 
+/* defines the different stages a send partitions must go through */
+enum MPIDIG_part_status_send_cc_t {
+    MPIDIG_PART_STATUS_SEND_READY,      /* 0 = partition is ready to be sent */
+    /* below insert all the needed operations */
+    MPIDIG_PART_STATUS_SEND_CTS,        /* CTS received */
+    /* the last value is the initialization value */
+    MPIDIG_PART_STATUS_SEND_INIT
+};
+
+/* defines the different stages a recv partitions must go through */
+enum MPIDIG_part_status_recv_cc_t {
+    MPIDIG_PART_STATUS_RECV_READY,      /* 0 = partition is received and ready */
```

Comment:
Suggest using "arrived" here to match the API.
```suggestion
    MPIDIG_PART_STATUS_RECV_ARRIVED,      /* 0 = partition arrived */
```

Reply:
solved in 87ccea181bd6cf17abb739abcd610e17782195f4